runtimes/common/src/benchmarks/xcm.rs

```diff
@@ -0,0 +1,48 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2024 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use cumulus_primitives_core::ParaId;
+use frame_support::parameter_types;
+use polkadot_runtime_common::xcm_sender::{NoPriceForMessageDelivery, ToParachainDeliveryHelper};
+use xcm::v4::prelude::*;
+
+use crate::constants::KILT;
+
+parameter_types! {
+	pub const RandomParaId: ParaId = ParaId::new(42424242);
+	pub ExistentialDepositAsset: Option<Asset> = Some((
+		Here,
+		KILT
```

Comment:
I think we can and should use the `ED` constant from common.

Reply:
Updated in: [d066dcc](https://github.com/KILTprotocol/kilt-node/pull/655/commits/d066dccff1fb279e1c06528957c94efa1a6de8e1)